mcs/build/profiles/monotouch.make

```diff
@@ -1,4 +1,6 @@
 include $(topdir)/build/profiles/monotouch_runtime.make
 
 PROFILE_MCS_FLAGS += \
-	-d:FULL_AOT_RUNTIME
+	-d:FULL_AOT_RUNTIME \
+	-d:MONO_SDK
```

Comment:
We try to do more FEATURE_ based approach nowadays, MONO_SDK is not in line with that